GlympsApp/Auth and Onboarding/BirthdayVC.swift

```diff
@@ -178,6 +180,9 @@ class BirthdayVC: UIViewController {
         }
     }
     
+    func logAmplitudeOnboardingStepSixOfNineCompleteBirthdayEvent() {
+        Amplitude.instance().logEvent("Onboarding Step 6/9 Complete (Birthday)")
```

Comment:
Think we should name these events something a little more searchable like `Onboarding Step Complete 6`. Info that we can just reference separately doesn't need to be in the event name